cfgov/teachers_digital_platform/models/activity_index_page.py

```diff
@@ -210,186 +208,14 @@ def dsl_search(self, request, *args, **kwargs):
-            if facet in request.GET and request.GET.get(facet):
-                selected_facets[facet] = [
-                    int(value) for value in request.GET.getlist(facet)
-                    if value.isdigit()
-                ]
-                facet_queries[facet] = facet + '_exact:' + (
-                    " OR " + facet + "_exact:").join(
-                    [str(value) for value in selected_facets[facet]]
-                )
-        payload = {
-            'search_query': search_query,
-            'results': [],
-            'total_results': 0,
-            'total_activities': total_activities,
-            'selected_facets': selected_facets,
-            'facet_queries': facet_queries,
-            'all_facets': {},
-        }
-
-        # Apply search query if it exists, but don't apply facets
-        if search_query:
-            sqs = sqs.filter(
-                content=search_query).order_by('-_score', '-date')
-        else:
-            sqs = sqs.order_by('-date')
-
-        # Get all facets and their counts
-        facet_counts = sqs.facet_counts()
-        all_facets = self.get_all_facets(
-            FACET_MAP, sqs, facet_counts, facet_queries, selected_facets
-        )
-
-        # List all facet blocks that need to be expanded
-        conditionally_expanded = {
-            facet_name for facet_name, facet_items in all_facets.items()
-            if any(
-                facet['selected'] is True for facet in facet_items
-            )
-        }
-        expanded_facets = ALWAYS_EXPANDED.union(set(conditionally_expanded))
-
-        payload.update({
-            'all_facets': all_facets,
-            'expanded_facets': expanded_facets,
-        })
-
-        # Apply all the active facet values to our search results
-        for facet_narrow_query in facet_queries.values():
-            sqs = sqs.narrow(facet_narrow_query)
-
-        results = [activity_card_map[activity.pk] for activity in sqs]
-        total_results = len(results)
-
-        payload.update({
-            'results': results,
-            'total_results': total_results,
-        })
-        self.results = payload
-        results_per_page = validate_results_per_page(request)
-        paginator = Paginator(payload['results'], results_per_page)
-        current_page = validate_page_number(request, paginator)
-        paginated_page = paginator.page(current_page)
-
-        context_update = {
-            'facets': all_facets,
-            'activities': paginated_page,
-            'total_results': total_results,
-            'results_per_page': results_per_page,
-            'current_page': current_page,
-            'paginator': paginator,
-            'show_filters': bool(facet_queries),
-        }
-        return context_update
-
     def get_context(self, request, *args, **kwargs):
         if not self.activity_setups:
             self.activity_setups = get_activity_setup()
-        if flag_enabled('ELASTICSEARCH_DSL_TDP'):
```

Comment:
Don't forget to remove the flag definition in `settings/base.py`.

Reply:
I thought we'd leave flag declarations alone until all apps are clear. Too cautious?

Reply:
I guess I'd say if you're removing the code that uses the flag, might as well remove the declaration.

Reply:
Sold.